dockernet/pkill.sh

```diff
@@ -0,0 +1,3 @@
+pkill -f "docker-compose .*simd.* logs" | true
+pkill -f "/bin/bash.*create_logs.sh" | true
+pkill -f "tail .*.log" | true
```

Comment:
Do we need logs? / Why do we need logs?

Reply:
I think Yes. 
Commands in a pkill.sh file via bash to prevent the exit code of 1 from stopping the make in Linux.
We need logs to make debugging easier during testing.